### ISSUE TITLE
version at keybase.io for non-mobile app

### ISSUE THREAD

**Issue by ra** [keybase/client#10848]
In the two app stores I can compare the available version with the installed version. I can run ```keybase version``` on my laptop. Where is the available version published for these installs?

**Comment by dabura667** [comment]
1. look at the version: ie `Client:  1.0.45-20180308194925+c80f6ae5ed`
2. Check the letters after the + `c80f6ae5ed`
3. Visit `https://github.com/keybase/client/commit/c80f6ae5ed` (replace the last part after the slash with your version.)
4. The commit says `zapu committed 2 days ago` so you know how long ago the version was made.
5. Also, you can look at https://github.com/keybase/client/commits/master to see when the most recent change was made (the top commit says 10 hours ago)
6. Also, the numbers before the + are the date that the version was created. `March 8th 2018 7:49:25 PM` = `20180308194925` and the "version" number is `1.0.45` which will usually only change based on tags here https://github.com/keybase/client/tags but keybase also stores its version number in `version.go` so looking at this file's history might help if they have forgotten to add a tagged version in the tags. https://github.com/keybase/client/blob/master/go/libkb/version.go

TL;DR

1. visit https://github.com/keybase/client/blob/master/go/libkb/version.go
2. check `keybase version` up til the first `-`

**Comment by ra** [comment]
thanks
